### PR TITLE
R doc 1604 dynamic object

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/4.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/Conventions.cs [hunk 1]
```diff
@@ -75,7 +75,7 @@ public void Examples()
                     PropertyNameConverter = mi => FirstCharToLower(mi.Name)
                     #endregion
                     ,
-                    #region TopologyCacheLocation
+                    #region AddIdFieldToDynamicObjects
```
Comment (maximburyak) [inline Documentation/4.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/Conventions.cs]: I think that you should keep the TopologyCacheLocation region

Comment (morhilai) [reply Documentation/4.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/Conventions.cs]: It's still there, lines 69-71. I think a typo I made and then fixed shows up in the commit.

Comment (maximburyak) [reply Documentation/4.2/Samples/csharp/Raven.Documentation.Samples/ClientApi/Configuration/Conventions.cs]: right

**Comment by maximburyak** [comment]
LGTM

**Pull request merged**